libqtile/widget/volume.py

```diff
@@ -100,31 +100,31 @@ def create_amixer_command(self, *args):
     def button_press(self, x, y, button):
         if button == BUTTON_DOWN:
             if self.volume_down_command is not None:
-                subprocess.call(self.volume_down_command)
+                subprocess.call(self.volume_down_command, shell=True)
```

Comment:
Please rebase on the master branch so that those changes don't appear here. :wink: